### DISCUSSION TITLE
Quais cursos escolher ?

### DISCUSSION THREAD

**Discussion by Jorge-Mamani** [frontendbr/forum#2431] [Perguntas e Respostas]
Olá pessoal, estou entrando na área de desenvolvimento , e hoje tenho noções básicas em Javascript , Html e CSS, e queria saber quais cursos recomendam, tendo em vista várias plataformas. Udemy , Alura Rocketset...
E queria saber a Experiência que tiveram .

**Comment by nazarepiedady** [comment]
@Jorge-Mamani o que tens por noções básicas de HTML, CSS e JavaScript?

Quais são os teus objetivos com o Frontend?

Quais ferramentas tencionas usar?

Quanto tempo tens disponível diariamente para dedicar aos estudos?

Tens a certeza que é o momento para começar a gostar dinheiro com cursos, enquanto existem bons conteúdos gratuitos na YouTube?

Já sabes o Inglês fluente?

As respostas que tiveres para estas questões dar-te-ão um norte.

**Comment by lucas-0liveira** [comment]
boa noite Jorge, recomendo você fazer um curso a curto prazo na udemy, lá na plataforma tem uma lista imensa de vários cursos para você aprender, indico a você que tem noção básica em javascript, html e css, aprimorar essas linguagens!

**Comment by jeffersonalionco** [comment]
@Jorge-Mamani Bom dia! Olha minha opnião. Você pode começar a fazer um curso de NodeJs, isso íra te preparar para a parte Back-end dos seus projetos como sites. como você tem nossão do Front-End então isso facilitara a sua vida.

Lembrando é uma dica que eu faria, mais as possibilidades são muitas porém vai depender do que você deseja no futuro como profissional.
